tasks/transifex-resjson-tasks.js

```diff
@@ -720,7 +720,7 @@ module.exports = function (grunt) {
         undefined if the code isn't of correct format.
     */
     function mapToTxLangCode(str) {
-        var matcher = str.match(/([a-z]{2})-([A-Z]{2}|latn)$/i);
+        var matcher = str.match(/([a-z]{2,3})(?:[-_]([a-z]{2}|latn))?$/i);
```

Comment:
Accepting the underscore will break things. When doing the mapping back from Transifex language codes to the file system,  Transifex's code `en_US` will be converted to `en-US` by `mapFromTxLangCode()` even though it originally was `en_US`. Accepting it would require additional changes for handling the mappings, so better leave it out here.

Otherwise happy to merge this in, after that small fix!